### PR TITLE
Unit test on queue jobs of CI flow. #5293

### PR DESCRIPTION
**Pull request by stevenc987**
*Issue #, if available:*

*Description of changes:* issue #5293
Make namex-api-ci works.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: services/namex-pay/setup.cfg [hunk 1]
```diff
@@ -75,7 +75,7 @@ max_line_length=120
 notes=FIXME,XXX,TODO
 ignored-modules=flask_sqlalchemy,sqlalchemy,SQLAlchemy,alembic,scoped_session
 ignored-classes=scoped_session
-disable=C0301,W0511,R0801,R0902
+disable=C0301,W0511,R0801,R0902,C0209
```
Comment (pwei1018) [inline services/namex-pay/setup.cfg:78]: I think we should not disable this check. We want to use f'{}' format in our code.

File: .github/workflows/namex-pay-ci.yml [hunk 2]
```diff
@@ -1,8 +1,9 @@
-name: Namex API CI
+name: Namex Pay CI
 
 on:
-  pull_request:
-    types: [assigned, synchronize]
+  push:
```
Comment (pwei1018) [inline .github/workflows/namex-pay-ci.yml:4]: Need pull request to trigger the CI flow.

File: services/namex-pay/requirements.txt [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
+itsdangerous==1.1.0
 jsonschema==3.2.0
 python-dotenv==0.17.1
 sentry-sdk[flask]
-aiohttp==3.7.4.post0
 asyncio-nats-client==0.11.4
 asyncio-nats-streaming==0.4.0
 pycountry==20.7.3
-Werkzeug==2.0.1
+Werkzeug==1.0.1
```
Comment (pwei1018) [inline services/namex-pay/requirements.txt:9]: The version is downgrade. Need to verify it.

File: services/namex-pay/tests/conftest.py [hunk 4]
```diff
@@ -58,12 +60,11 @@ def utcnow(cls):
...
     """Return a session-wide application configured in TEST mode."""
-    # _app = create_app('testing')
-    _app = Flask('testing')
+    _app = Flask('testing')    
     print(config)
     _app.config.from_object(get_named_config('testing'))
-    db.init_app(_app)
-    queue.init_app(_app, asyncio.new_event_loop())
+    _db.init_app(_app)
+    # queue.init_app(_app, asyncio.new_event_loop())
```
Comment (pwei1018) [inline services/namex-pay/tests/conftest.py:67]: Does this need to get disable?

**Review by pwei1018 — APPROVED** [review]

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**